students/AlyssaHong/lesson03_1/slicing.py

```diff
@@ -0,0 +1,64 @@
+"""
+Author: Alyssa Hong
+Date: 10/22/2018
+Lesson3 Assignments > Slicing Lab Exercise
+"""
+#Get the basics of sequence slicing downself.
+
+#1 with the first and last items exchanged.
+def exchange_first_last(seq):
+    a_new_sequence = seq[-1], seq[0]
+    seq[0], seq[-1] = a_new_sequence
+    print("result #1:", seq)
+    return exchange_first_last
+
+#2 with every other item removed.
+def every_other_item_remove(seq):
+    count_seq = len(seq)
+    del seq[1:int(count_seq-1)]
+    print("result #2:", seq)
+    return every_other_item_remove
+
+#3 with the first 4 and the last 4 items removed, and then every other item
+# in between.
+def select_item_and_remove(seq):
+    # print(seq[4:-4])
+    del seq[3]
+    del seq[-4]
+    del seq[4:-3]
+    print("result #3:", seq)
+    return select_item_and_remove
```

Comment:
This one too - if you pass in 1,2,3,4,5,6,7,8,9,10 you would expect 1-4 to be removed, as well as 7-10, then return every other between the 2.  So, the return would be 5